docs/modules/casper.rst

```diff
@@ -1357,16 +1368,16 @@ Retrieves current page title::
 ``mouseEvent()``
 -------------------------------------------------------------------------------
 
-**Signature:** ``mouseEvent(String type, String selector)``
+**Signature:** ``mouseEvent(String type, String selector, [Number|String X, Number|String Y])``
 
 .. versionadded:: 0.6.9
 
 Triggers a mouse event on the first element found matching the provided selector.
 
-Supported events are ``mouseup``, ``mousedown``, ``click``, ``mousemove``, ``mouseover`` and ``mouseout``::
```

Comment:
imo you should add a new "versionadded" section, and talk about phantomjs <= 1.9.8 limitations